### ISSUE TITLE
Lat/lon truncated to 2 decimal places

### ISSUE THREAD

**Issue by javier-molina** [AtlasOfLivingAustralia/extended-data-model#55]
This below needs to be investigated

The following map looks dodgy when I zoom in. It looks like the lats/lons have been truncated to 2 decimal places (ie about 1 kilometre) when at least 4 should be used (ie 10 metres)

  
![Screen Shot 2022-09-26 at 3 45 34 pm](https://user-images.githubusercontent.com/8326394/192202365-14627d2c-b148-4df0-a8a8-46c322be2de7.png)

**Comment by qifeng-bai** [comment]
Suspect the lat/lng were truncated on the Pipelines

https://events-test.ala.org.au/graphql?queryId=d3795e58c90d327255ef22c69eab28f3c913e8b0&strict=true&variables=%7B%22locationID%22:%2224.11S,+152.71E%22%7D

**Comment by javier-molina** [comment]
TL;DR

This is a data issue not a coding issue.


Coordinates are rounded from source, see occurrence extract from  [dr9942-dwca-imos_atf_ad-v1.8.zip](https://csiroau.sharepoint.com/:u:/r/sites/ALA/Shared%20Documents/General/Analysis/Datasets/[dr9942-dwca-imos_atf_ad-v1.8.zip](https://csiroau.sharepoint.com/:u:/r/sites/ALA/Shared%20Documents/General/Analysis/Datasets/dr9942-dwca-imos_atf_ad-v1.8.zip?csf=1&web=1&e=3L5zk0)?csf=1&web=1&e=3L5zk0)


![Screen Shot 2022-12-15 at 2 46 37 pm](https://user-images.githubusercontent.com/8326394/207768441-e1b09948-7be7-4526-9ef9-ea38e2bea44d.png)


While it makes sense to deploy sensors in a planned area, hence the visible grids, that does not explain why the 2 digit rounding. I can only speculate it is some form of obfuscation to hide the actual location of the sensors. 

@charvolant would you mind casting your eyes to this issue to confirm we are satisfied with the dataset as is or otherwise you think there is any additional work to source a refreshed version (in the future)?

**Comment by javier-molina** [comment]
From stand up meeting on 10/01/23 Doug confirms this is a data issue
